doc/source/release_notes.rst

```diff
@@ -2,6 +2,39 @@
  Release History
 =================
 
+v1.3.1 (2019-01-03)
+===================
+
+Features
+--------
+
+* Add :class:`~ophyd.FakeEpicsSignalWithRBV`, which is to
+  :class:`~ophyd.FakeEpicsSignal` as :class:`~ophyd.EpicsSignalWithRBV` is to
+  :class:`~ophyd.EpicsSignal`.
+* Add enum-spoofing to :class:`~ophyd.FakeEpicsSignal`.
+* A default handler is added to the ``'ophyd'`` logger at import time. A new
+  convenience function, :func:`~ophyd.set_handler`, addresses common cases
+  such as directing the log output to a file.
+
+Bug Fixes
+---------
+
+* Always interpret motor positions as floats, even if set to an integer
```

Comment:
I think this only applies to simulated motors (https://github.com/NSLS-II/ophyd/pull/606/files) the PR title was overly dramatic (sorry).